tests/tasks/setup_candlepin.yml

```diff
@@ -6,56 +6,32 @@
 - name: Deploy Candlepin
   when: lsr_rhc_test_data_file | length == 0
   block:
-    - name: Skip on OSes not supported by Candlepin
-      meta: end_play
-      when:
-        - ansible_distribution in ["CentOS", "RedHat"]
-        - ansible_distribution_major_version | int > 8
-
     - name: Set helper fact for Candlepin base URL
       set_fact:
         _cp_url: "https://{{ lsr_rhc_test_data.candlepin_host }}:{{ lsr_rhc_test_data.candlepin_port }}{{ lsr_rhc_test_data.candlepin_prefix }}"  # yamllint disable-line
 
-    - name: Clone ansible-role-candlepin repo
-      git:
-        repo: https://github.com/candlepin/ansible-role-candlepin.git
-        # wokeignore:rule=master
-        version: master
-        dest: "{{ playbook_dir }}/roles/candlepin"
-      delegate_to: 127.0.0.1
-      become: false
-
-    - name: Create 'candlepin' user
-      user:
-        name: candlepin
-
-    - name: Create sudoers configuration
-      copy:
-        dest: /etc/sudoers.d/candlepin
-        content: "candlepin ALL=(ALL) NOPASSWD: ALL\n"
-        mode: 0440
+    - name: Add candlepin.local to /etc/hosts
+      lineinfile:
+        path: /etc/hosts
+        line: "127.0.0.1   candlepin.local"
+        regexp: ".*candlepin.local"
 
     - name: Install needed packages
       package:
         name:
-          - expect
+          - podman
         state: present
 
-    - name: Run Candlepin install
-      block:
-        - name: Include candlepin role
-          include_role:
-            name: candlepin
-          vars:
-            cp_configure_mariadb: false
-            cp_configure_ruby: false
-            cp_deploy_args: "-g -a -f -t -r"
-            cp_git_ref: 9263a30b1612a22ec00ae05bd7d9ed6e0cdedbaf
-
-    - name: Enable tomcat
-      service:
-        name: tomcat
-        enabled: true
+    - name: Start Candlepin container
+      containers.podman.podman_container:
```

Comment:
Are we going to use this for downstream testing?  If so, I would rather not have another test dependency on an external, community supported collection.

Reply:
it's used only when not using an external server (see `LSR_RHC_TEST_DATA` in `tests/README.md`); considering I manually call `podman` anyway, I guess I can avoid the usage of the collection and run `podman run` manually